modules/rosa-aws-customer-managed-policies.adoc

```diff
@@ -0,0 +1,21 @@
+// Module included in the following assemblies:
+//
+// * rosa_architecture/rosa-sts-about-iam-resources.adoc
+
+[id="rosa-aws-customer-managed-policies_{context}"]
+= Customer-managed policies
+ROSA users are able to attach customer-managed policies to the IAM roles required to run and maintain both ROSA (classic architecture) and ROSA clusters. This capability is not uncommon with AWS IAM roles.
```

Comment:
Should the specified range of clusters here be "ROSA classic clusters and ROSA hosted control plane clusters" here instead?

Reply:
Hello @davidleerh, well that is a good point...BUT...the official naming is now as I have written it out.....given that, it is going to take awhile for all the docs to reflect that, but that is what the powers that be wanted so there you have it....good catch, though!